docs/tinlake/overview/introduction.md

```diff
@@ -7,46 +7,18 @@ redirect_from:
- 
- ### NFT Assets 
- In Tinlake the collateral generally is in form of an NFT. Though with a custom adapter, any other collateral could be used. For an asset to be used as collateral in Tinlake it generally must satisfy two conditions:
-  
-  1. Ownership of the asset must be transferable to the Tinlake `Shelf` which holds it in escrow until the loan is repaid
-  2. Appraisal: There must be a way to determine the value of the collateral with oracles or automated pricing models.
+Each Collateral Value Token (CVT) represents a fraction of the respective set of collateral, and the number of minted CVTs will be determined by independent configurations for each different type of assets.  As borrowers deposit NFTs into Tinlake, new CVT is minted on demand to draw funding from crypto funding sources.
 
+Collateral management (an administrator) assumes the responsibilities of underwriting and valuing the assets, as well as keeping a balanced portfolio. In the case of payment default, collateral governance manages the liquidation of the assets.
 
- *Add overview graphic of Tinlake*
+## User roles
+### Borrowers and Lenders
+Tinlake interacts with borrowers and lenders. Generally, borrowers have individual loans with varying terms and varying durations drawn against their collateral. Lenders invest in the entire collateral portfolio at a defined rate. Borrowers can borrow by depositing an NFT collateral representing ownership of an asset, which they can get back upon repaying their debt.
 
- ## Contracts Overview
- | Contract | Description |
- |----------|-------------|
- | Title | Tracks ownership of each loan. Implements the NFT interface so loans themselves become NFTs. ||
- | Lightswitch | Triggers settlement of the tinlake system if the value of the collateral falls below a certain treshold or some other condition is met(e.g the MakerDAO CDP gets bitten) |
- | Pile | Manages loan debt and token balance, pays out to borrower and receives repayments | 
- | Shelf | Keeps track of approved NFTs and owns NFTs |
- | Valve | Controls minting and burning of Collateral Value Token |
- | Collateral | The collateral value token. |
- | Appraiser| Appraiser is a module that can be plugged in that returns the value of the collateral. | 
- | Reception | Wrapper contract for borrowers |
- | Desk | The "trading desk", wrapper contract for the administrator to manage the lender side |
- | MakerAdapter | Manages a CDP |
- | BackerAdapter | Uses a single account to dra
- | Deployer | Deploys all components in one transaction setting the right wards |
- 
+### Administrator
+Generally, an administrator determines what assets borrowers are allowed to bring in as collateral and how much they are allowed to borrow against the assets. This administrator could be a single centralized entity, but could be a set of automated smart contracts automatically qualifying and pricing an asset. Or, you could plug in a DAO that takes control of the Tinlake deployment.
```

Comment:
... This administrator could be a single centralized entity, a set of smart contracts automatically qualifying and pricing an asset or a decentralised autonomous organisation (DAO) that manages the Tinlake deployment.